sdk/communication/Azure.Communication.Common/src/CommunicationTokenRefreshOptions.cs

```diff
@@ -13,20 +13,27 @@ namespace Azure.Communication
     /// </summary>
     public class CommunicationTokenRefreshOptions
     {
-        internal bool RefreshProactively { get; }
-        internal Func<CancellationToken, string> TokenRefresher { get; }
-
+        /// <summary>Default time span in minutes before token expiry that causes the 'tokenRefresher' to be called.</summary>
+        internal const int DefaultExpiringOffsetMinutes = 10;
         /// <summary>The initial token.</summary>
         public string InitialToken { get; set; }
-
-        private Func<CancellationToken, ValueTask<string>> _asyncTokenRefresher;
-
         /// <summary>The asynchronous token refresher.</summary>
         public Func<CancellationToken, ValueTask<string>> AsyncTokenRefresher
         {
             get => _asyncTokenRefresher = _asyncTokenRefresher ?? (cancellationToken => new ValueTask<string>(TokenRefresher(cancellationToken)));
             set => _asyncTokenRefresher = value;
         }
+
+        /// <summary>The time span before token expiry that causes the 'tokenRefresher' to be called if 'refreshProactively' is true.</summary>
+        public TimeSpan RefreshTimeBeforeTokenExpiry { get; set; } = new TimeSpan(0, 0, DefaultExpiringOffsetMinutes, 0);
```

Comment:
please rename all occurrences of `RefreshTimeBeforeTokenExpiry` to `RefreshIntervalBeforeTokenExpiry`